src/main/java/programminglife/gui/controller/GuiController.java

```diff
@@ -162,8 +161,9 @@ private void initRecent() {
         try {
             Files.createFile(new File("Recent.txt").toPath());
         } catch (FileAlreadyExistsException e) {
+            Alerts.warning("File already exists, this is a good thing!");
```

Comment:
Dont think this is a correct way to catch an exception. 
This would be as if you just did 1+1 and then said its good like this.